app/controllers/gobierto_admin/gobierto_people/people_controller.rb

```diff
@@ -6,7 +6,7 @@ class PeopleController < BaseController
       before_action { module_enabled!(current_site, "GobiertoPeople") }
 
       def index
-        @people = current_site.people.sorted
+        @people = current_site.people.includes(:events, :statements, :posts).sorted
```

Comment:
I'm refactoring this thing a little bit because it may be a performance killer. Let me please merge this one once is approved 🙏